src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -522,7 +520,7 @@ private void RegisterResolversManifests(ElementLocation location)
                 // The collections are never modified after this point.
                 // So I've made them ReadOnly
                 _specificResolversManifestsRegistry = specificResolversManifestsRegistry.AsReadOnly();
```

Comment:
Why this `AsReadOnly()` is left and next one  is removed?

Reply:
Updated the other one as well.